server/config/passport.js

```diff
@@ -13,6 +13,9 @@ import User from '../models/user';
 const accountSuspensionMessage =
   'Account has been suspended. Please contact privacy@p5js.org if you believe this is an error.';
 
+const accountLinkingErrorMessage =
+  'Account is already linked to another account.';
+
```

Comment:
Thanks for your work on this.  I'm going to close this PR and go with #3044 instead because I don't like this particular change.  IMO we want to keep the name of the service in there.